problems/sumDigits.yaml

```
@@ -0,0 +1,32 @@
+id: 3
+title: SumDigits
+text: |
+  Write a function called sumDigits that takes in a non-negative ineteger,
+  given an input num, and returns that sum of all its digits. If the number
+  is negative, the first digit should count as the only negative.
```

```suggestion
  Write a function called sumDigits that takes in an integer and returns the 
  sum of all its digits. If the number is negative, the first digit should count as 
  the only negative.
```
this reads more clearly to me :)

---

Thanks. Making changes now